core/src/main/scala/org/apache/spark/util/collection/ExternalAppendOnlyMap.scala

```diff
@@ -49,18 +48,25 @@ import org.apache.spark.executor.ShuffleWriteMetrics
  * writes. This may lead to a performance regression compared to the normal case of using the
  * non-spilling AppendOnlyMap.
  */
-@DeveloperApi
```

Comment:
Yeah. The issue is that Spillable now requires a TaskMemoryManager, which won't be available on the driver. I'm pretty sure that this class was never intended to be DeveloperAPI'd. If you feel strongly that we shouldn't break this, I can add a compatibility test.